coffeecard/CoffeeCard.Library/Services/AccountService.cs

```diff
@@ -289,21 +307,38 @@ private async Task AnonymizeUser(User user)
             user.DateUpdated = DateTime.Now;
             user.PrivacyActivated = true;
             user.UserState = UserState.Deleted;
-            await _context.SaveChangesAsync();            
+            await _context.SaveChangesAsync();
         }
-        
-        private User VerifyTokenClaimAndUser(string token)
+
+        private async Task<User> VerifyTokenClaimAndUserAsync(string token)
```

Comment:
Not on you, but doesn't it seem a bit weird that we read claims in AccountService and not in the TokenService?

Reply:
I agree, I suppose the original reasoning would have been it returns a user object, hence its placement in accountService. I suppose we could move it to the tokenSerive as a getEmailFromVerificationToken, or something like that. Then we could still let the accountService handle the retrieval of the suer object.

Reply:
You can decide if you want to do anything about this or not :)

Reply:
Moved to TokenService